build/references/coreth-arc20s.md

```diff
@@ -0,0 +1,108 @@
+# Coreth ANTs and ARC20s
+
+## ANTs on the C Chain
+
+### Why move an ANT from the X Chain to the C Chain?
+
+An Avalanche Native Token (ANT) is any token created on the X Chain. These tokens can be exchanged on the X Chain at lightning fast speeds taking advantage of the superior performance of using a DAG instead of a linear chain as the basis for the blockchain.
+
+When we want to use smart contract functionality, we need a total ordering of state transitions (transactions), so that each transaction can modify the state of the network without having to worry about external dependencies.
+
+Therefore, when we want to use smart contract functionality, we must perform an atomic swap to move ANTs from the X Chain to the C Chain.
+
+### ANTs on the C Chain
+
+#### Native Coin - AVAX
+
+ANTs on the C Chain are either AVAX - the Avalanche native asset - or another asset that was created on the X Chain.
```

Comment:
Use proper `em` dashes w/ `&emdash;`

`ANTs on the C Chain are either AVAX&mdash;the Avalanche native asset&mdash;or another asset that was created on the X Chain.`

ANTs on the C Chain are either AVAX&mdash;the Avalanche native asset&mdash;or another asset that was created on the X Chain.

Reply:
Updated